internal/controllers/terraformpullrequest/layer.go

```diff
@@ -106,6 +106,15 @@ func layerFilesHaveChanged(layer configv1alpha1.TerraformLayer, changedFiles []s
 		if strings.Contains(f, layer.Spec.Path) {
 			return true
 		}
+		// Check if the file is under an additionnal trigger path
+		if val, ok := layer.Annotations[annotations.AdditionnalTriggerPaths]; ok {
+			for _, p := range strings.Split(val, ",") {
+				p = ensureAbsPath(p)
```

Comment:
@corrieriluca I think we're going to need the same fix you did for the webhook here

Reply:
Resolved with the refactor introduced in https://github.com/padok-team/burrito/pull/129/commits/1a663de3b5f917ffc326b960d57e0188e255f3c8